tests/reftests/unhelpful-conflicts-002.test

```diff
@@ -28,40 +28,7 @@ ocaml-config is now pinned locally (version 1)
 ### opam install --show expect.0.0.6
 [ERROR] Package conflict!
   * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.00.0
-    not available because the package is pinned to version 4.12.0
-  * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.01.0
-    not available because the package is pinned to version 4.12.0
-  * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.02.0
-    not available because the package is pinned to version 4.12.0
-  * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.03.0
-    not available because the package is pinned to version 4.12.0
-  * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.04.0
-    not available because the package is pinned to version 4.12.0
-  * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.05.0
-    not available because the package is pinned to version 4.12.0
-  * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.06.0
-    not available because the package is pinned to version 4.12.0
-  * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.07.0
-    not available because the package is pinned to version 4.12.0
-  * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.08.0
-    not available because the package is pinned to version 4.12.0
-  * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.10.0
-    not available because the package is pinned to version 4.12.0
-  * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.11.0
-    not available because the package is pinned to version 4.12.0
-  * Missing dependency:
-    - expect >= 0.0.6 -> batteries -> ocaml < 4.12.0
+    - ocaml < 4.12.0
```

Comment:
For these conflict message tests, it can be helpful to know what is the "intended output". I'm not sure if it is always simple to know in advance.
Here if we know that it is `ocaml < 4.12.0` the conflict that we want to highlight (i suppose that from the pin above), a comment may help to know that the fix is good (and not a simplification that outputs the wrong message).
This case is simple thought to determine thought.